### PR TITLE
Integration tests for live blogs

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What?
This PR adds some Cypress specs for testing interactivity on liveblogs

### Skipped second page test
This test is present but currently skipped pending [another PR](https://github.com/guardian/dotcom-rendering/pull/4099)

### But isn't this blog going to be dead?
We use `?live=true` to force the blog to be 'live' to the level where our client side polling code will execute.

### CODE REVIEW

File: dotcom-rendering/cypress/integration/parallel-3/liveblog.interactivity.spec.js [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+			.first()
+			.should('have.attr', 'data-gu-ready', 'true');
+		cy.scrollTo('center');
+		cy.get(`[data-cy="toast"]`).should('not.exist');
+		cy.window().then(function (win) {
+			win.mockLiveUpdate({
+				numNewBlocks: 1,
+				html: '<p>New block</p>',
+				mostRecentBlockId: 'abc',
+			});
```
Comment (mxdvl) [inline dotcom-rendering/cypress/integration/parallel-3/liveblog.interactivity.spec.js:55]: I think this is the right approach, but highlighting here for others to see.

**Review by mxdvl — COMMENTED** [review]
Is there a reason we do not use fat arrow functions? Is the scope of `this` important in this case?

EDIT: I checked and this in in line with other Cypress tests

File: dotcom-rendering/cypress/integration/parallel-3/liveblog.interactivity.spec.js [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+const ignoreTwitterError = () => {
+	cy.on('uncaught:exception', (err) => {
+		if (err.message.includes('Illegal invocation')) {
+			// Thrown by the twitter lib. This error is unrelated to the test in question
+			// so return false to prevent this error from failing this test
+			return false;
+		}
+	});
+};
```
Comment (mxdvl) [inline dotcom-rendering/cypress/integration/parallel-3/liveblog.interactivity.spec.js:22]: Are we sure it's fine to ignore this error?

Could we extract this into a JSDoc style comment to the const?

**Review by mxdvl — APPROVED** [review]
More tests: nice!

File: dotcom-rendering/cypress/integration/parallel-3/liveblog.interactivity.spec.js (hunk 2 above)
Comment (oliverlloyd) [reply dotcom-rendering/cypress/integration/parallel-3/liveblog.interactivity.spec.js]: It's conceivable that there is an issue but I'd rather solve problems that get surfaced on the real site, not inside the strange Cypress world where globals like `twttr` can have issues. Plus we're testing that the function works so I'm happy to ignore. 

I've moved the comment into JSDoc 👍

**Pull request merged**